src/screens/Ticketing/Purchase/Confirmation/index.tsx

```diff
@@ -241,31 +328,73 @@ const Confirmation: React.FC<ConfirmationProps> = ({
           />
         ) : (
           <View>
-            <Button
-              color="secondary_1"
-              text={t(PurchaseConfirmationTexts.paymentButtonVipps.text)}
-              disabled={!!error}
-              accessibilityHint={t(
-                PurchaseConfirmationTexts.paymentButtonVipps.a11yHint,
-              )}
-              icon={Vipps}
-              iconPosition="left"
-              onPress={payWithVipps}
-              viewContainerStyle={styles.paymentButton}
-            />
-            {enableCreditCard && (
+            {previousMethod ? (
+              <View
+                style={{
+                  flex: 1,
+                  flexDirection: 'column',
+                }}
+              >
+                <Button
+                  text={getPaymentOptionTexts(previousMethod)}
+                  color="primary_2"
+                  disabled={!!error || !previousMethod}
+                  iconPosition="right"
+                  icon={
+                    previousMethod.paymentType === PaymentType.MasterCard
+                      ? MasterCard
+                      : previousMethod.paymentType === PaymentType.Vipps
+                      ? Vipps
+                      : Visa
+                  }
+                  viewContainerStyle={styles.paymentButton}
+                  onPress={() => {
+                    if (previousMethod) {
+                      selectPaymentOption(previousMethod);
+                    }
+                  }}
+                ></Button>
+                <TouchableOpacity
+                  style={{
+                    marginTop: 24,
+                  }}
+                  disabled={!!error}
+                  onPress={() => {
+                    selectPaymentMethod();
+                  }}
+                  accessibilityHint={t(
+                    PurchaseConfirmationTexts.choosePaymentOption.a11yHint,
+                  )}
+                >
+                  <View
+                    style={{
+                      flex: 1,
+                      flexDirection: 'row',
+                      justifyContent: 'center',
+                    }}
+                  >
+                    <ThemeText
+                      style={{
+                        fontWeight: '400',
+                        fontSize: 16,
+                      }}
```

Comment:
Her bør det brukes `type="body__primary--bold"` i stedet for å eksplisitt sette fontWeight eller fontSize